torch/quantization/fx/quantization_patterns.py

```diff
@@ -1239,6 +1261,13 @@ def __init__(
         elif node.op == "call_module":
             self.op = type(modules[str(node.target)])
 
+    def is_output_quantized(self, qconfig, is_reference):
```

Comment:
This looks like a dup of the same function earlier

Reply:
yes, and binary_op_supported_dtypes is replaced with default_op_supported_dtypes